app/services/online_codes_import_service.rb

```diff
@@ -0,0 +1,32 @@
+require 'csv'
+
+class OnlineCodesImportService
```

Comment:
It would be awesome if this class had some test coverage

Reply:
I agree, but acceptance criterias in this task do not mention about tests so I didn't know if I should go further :) Of course I can try implementing tests for that feature :)

Reply:
Acceptance criteria (given by a client) almost never mentions tests :) It's up to a developer to deliver them. 

In the case of a training course like this one, it's probably up to you whether you'd like to train that part of the job.

Reply:
OK! In that case I will implement tests :)